CoreVisualGraph/src/au/gov/asd/tac/constellation/graph/visual/concept/VisualConcept.java

```diff
@@ -82,12 +86,12 @@ public String getName() {
                 .build();
         public static final SchemaAttribute BLAZE_OPACITY = new SchemaAttribute.Builder(GraphElementType.GRAPH, FloatAttributeDescription.ATTRIBUTE_NAME, "blaze_opacity")
                 .setDescription("The opacity of blazes on the graph")
-                .setDefaultValue(1.0)
+                .setDefaultValue(((float) ((float)(PREFERENCES.getInt(GraphPreferenceKeys.BLAZE_OPACITY, GraphPreferenceKeys.BLAZE_OPACITY_DEFAULT)/100.0))))
```

Comment:
What's the reason behind the outside `float` cast? If NetBeans throws a warning perhaps try changing the division to `100f`. I don't have your code checked out so apologies if I've missed something obvious.

Reply:
See latest commit "Resolve PR changes" where I change from double casted floats, to division by a float value